config/agents.go

```diff
@@ -12,13 +12,15 @@ const (
 )
 
 type AgentConfig struct {
-	ID          string  `yaml:"id" json:"id"`
-	Image       string  `yaml:"image" json:"image"`
-	Manifest    string  `yaml:"manifest" json:"manifest"`
-	IsLocal     bool    `yaml:"isLocal" json:"isLocal"`
-	StartBlock  *uint64 `yaml:"startBlock" json:"startBlock,omitempty"`
-	StopBlock   *uint64 `yaml:"stopBlock" json:"stopBlock,omitempty"`
-	AlertConfig *protocol.AlertConfig
+	ID           string        `yaml:"id" json:"id"`
+	Image        string        `yaml:"image" json:"image"`
+	Manifest     string        `yaml:"manifest" json:"manifest"`
+	IsLocal      bool          `yaml:"isLocal" json:"isLocal"`
+	StartBlock   *uint64       `yaml:"startBlock" json:"startBlock,omitempty"`
+	StopBlock    *uint64       `yaml:"stopBlock" json:"stopBlock,omitempty"`
+	Redundancy   uint          `yaml:"redundancy" json:"redundancy"`
```

Comment:
I think this probably should be "ShardCount" or something similar.

Reply:
I think AgentConfig, also the bot's environment variables, should have both the `ShardCount` (local instance count), and the `Redundancy` (total number of instances globally)

Reply:
possibly fixed with 42a367b10fceaaba2fbcbb5bb19541e3c3ddbefa